src/fragments/lib/analytics/ios/record.mdx

```diff
@@ -43,6 +43,47 @@ Amplify.Analytics.flushEvents()
 
```

Comment:
Note for configuration sample above: need to verify whether the `autoFlushEventsInterval` is seconds or milliseconds. iOS sample shows seconds, Android sample shows milliseconds. That may be accurate, but wanted to double check

Reply:
This is not correct on iOS, the `autoFlushEventsInterval` option is read from inside the `awsPinpointAnalyticsPlugin` block as documented.
Also I can confirm iOS uses seconds.